tests/unit/test_exceptions.py

```diff
@@ -33,3 +33,6 @@ def test_all_validation_errors_are_validation_errors(self):
     def test_invalid_option_is_validation_error(self):
         assert isinstance(exceptions.InvalidOption('msg'),
                           exceptions.ValidationError)
+
+    def test_betamaxerror_repr(self):
+        assert repr(exceptions.BetamaxError('test')) == "BetamaxError('test',)"
```

Comment:
Sorry. I didn't catch your comment that you were adding a test. I've found testing `repr` to be a really bad idea. As soon as the object changes (and the `repr` changes) your tests break and for no really good reason either.

Reply:
What about keeping it without the assert? As a regression test to make sure there's no exception?

Reply:
You mean just?

```py
def test_betamaxerror_repr(self):
    repr(exceptions.BetamaxError('test'))
```

Reply:
well maybe with a comment, but yes. currently the test would fail (actually error).

Reply:
or even test it for all of them, similarly to https://github.com/betamaxpy/betamax/pull/141

Reply:
I think that everything inherits from `BetamaxError`. If you want to test that none of them raise exceptions then that's fine. I think we should parametrize the test though instead of using a for-loop